wallet/datastore.go

```diff
@@ -364,6 +364,7 @@ func (pg *Postgres) LinkWallet(ID string, userDepositDestination string, provide
 
 	err = pg.TxLinkWalletInfo(tx, ID, userDepositDestination, providerLinkingID, anonymousAddress, depositProvider)
 	if err != nil {
+		fmt.Println("!!!! err: ", err)
```

Comment:
we can remove this no?

Reply:
updated in https://github.com/brave-intl/bat-go/pull/541/commits/4a6999e3d9af426bc5ac753a8fa816662c03bea0